oteapi/strategies/mapping/mapping.py

```diff
@@ -0,0 +1,63 @@
+"""Mapping filter strategy."""
+# pylint: disable=unused-argument
+from typing import TYPE_CHECKING
+
+from pydantic.dataclasses import Field, dataclass
+
+from oteapi.models import MappingConfig, SessionUpdate
+
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Any, Dict, Optional
+
+
+class MappingStrategyConfig(MappingConfig):
+    """Strategy-specific mapping config."""
+
+    mappingType: str = Field(
+        "triples",
+        description="Mapping type.  Normally this is just 'mapping'.",
+    )
```

Comment:
No need for this - the `mappingType` is set through the `create_strategy()` call, just like any other strategy.
```suggestion
```